### PR TITLE
Update SE.yml

### PR DESCRIPTION
**Pull request by gabriel-seow**

### CODE REVIEW

File: public/data/travel/SE.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 ---
-reopening:
-  - 'Only travelers originating from the EEA are allowed entry. Other foreign nationals are [not allowed entry until October 31, 2020](https://www.thelocal.se/20200827/sweden-extends-travel-ban-for-non-eu-residents-october).'
```
Comment (jannwong) [inline public/data/travel/SE.yml:3]: i'm confused. why are we removing this? we just need to add 2020-10-31?

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comment

File: public/data/travel/SE.yml (hunk 1 above)
Comment (gabriel-seow) [reply public/data/travel/SE.yml:3]: Apologies, will add it in

**Review by jannwong — APPROVED** [review]

**Pull request merged**